index.es6

```
@@ -119,26 +127,10 @@ export default class GoogleSearch extends React.Component {
         url: src,
         type: 'script',
       }).catch((e) => {
-        // Let provide a fallback if we can't load the GCS script.
-        const fallbackHTML = `
-            <form acceptCharset="UTF-8" method="GET"
-              id="search-theme-form" action="${this.props.resultsUrl}"
-              class="gsc-input"
-            >
-              <input
-                type="text" maxLength="128" name="${this.props.queryParameterName}"
-                id="edit-search-theme-form-1"
-                value=""
-                title="Enter the terms you wish to search for."
-                class="gsc-input"
-              />
-              <input type="hidden" name="cx"
-                value="${this.props.cx}" id="edit-cx"
-              />
-            </form>`;
         this.setState({
-          fallbackHTML,
+          useFallback: true,
         });
+        this.focusSearchField();
         console.error('An error occurs loading or executing Google Custom Search: ', e.message);
```

You need to rethrow the error here, otherwise this turns into a resolved promise.